src/components/10-atoms/progress-bar/index.scss

```diff
@@ -0,0 +1,45 @@
+:host {
+  display: block;
+}
+
+.a-progress-bar {
+  display: block;
+  
+  &__progress-bar {
+    border-radius: 10px;
+    background-color: $color-ocean-blue;
+    height: 12px;
+  }
+
+  &__border {
+    width: 100%;
+    border-radius: 10px;
+    background-color: $color-mercury;
+  }
+
+  &__small {
+    height: 8px;
+  }
+
+  &__no-border-radius {
+    border-radius: 0px;
+
+    &-loader {
+      border-radius: 0px 10px 10px 0px;
+
+      &-max {
+        border-radius: 0px;
```

Comment:
This is a weird hierarchy.

1st level: You set it to 0
2nd level: You set it to something else
3rd level: You set it to 0 again

Reply:
1st level: the background bar (grey bar)
2st level: the main loading bar (blue bar)
3st level: removes the radius on the right side of the main bar when its at 100%

But this will change now anyways